gui-libs/tepl/tepl-6.2.0.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org meson virtualx
+
+DESCRIPTION="GtkSourceView-based text editors and IDE helper library"
+HOMEPAGE="https://gitlab.gnome.org/swilmet/tepl"
+
+LICENSE="LGPL-3+"
+SLOT="6"
```

Comment:
Looks like the SOVERSION changed from 6.0.2 to 6.2.0:

````diff
-/usr/lib64/libtepl-6.so.0.0.0
+/usr/lib64/libtepl-6.so.1.0.0
````

I think we need a subslot and to update reverse dependencies to bind to it.